test/case_sensitive_measurable_test.rb

```diff
@@ -1,217 +0,0 @@
-require "test_helper"
```

Comment:
This test wasn't really testing much of the case sensitive behaviours of the original class, so I opted to stick with the existing case sensitive tests in `conversion_test.rb`